src/weekly/query.js

```diff
@@ -123,7 +123,7 @@ async function queryZuora (deliveryDate, config: Config) {
        (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
        ProductRatePlanCharge.ProductType__c = 'Adjustment' AND
        RateplanCharge.Name = 'Holiday Credit' AND
-       RatePlanCharge.EffectiveStartDate <= '${formattedDeliveryDate}' AND
+       RatePlanCharge.HolidayStart__c <= '${formattedDeliveryDate}' AND
```

Comment:
👍 Doh, I thought we'd already done this! I remember that we were going to wait for all scheduled v1 holidays to finish first, and the reminder ticket must have just got lost in a Trello backlog!

Reply:
Ah, that was home delivery.

Reply:
On page 5 of the runbook it says:

> b) Select a Holiday Start date which matches the Friday of the first publication the customer does not want to receive (note they may typically receive publications a day-or-two early)

So it looks like it should be safe.

Reply:
I'll check for how many cases have an effective start date and no holiday start date.